grepper_python/main.py

```diff
@@ -0,0 +1,156 @@
+"""
+The MIT License
+
+Copyright (c) 2010-2023 Grepper, Inc. (https://www.grepper.com)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
+
+from __future__ import annotations
+
+from typing import Optional, List
+
+from .answer import GrepperAnswer
+from .exceptions import BadRequest
+from .exceptions import Unauthorized
+from .exceptions import Forbidden
+from .exceptions import NotFound
+from .exceptions import MethodNotAllowed
+from .exceptions import TooManyRequests
+from .exceptions import InternalServerError
+from .exceptions import ServiceUnavailable
```

Comment:
You are still using relative imports instead of absolute imports

Reply:
oh yeah thanks for pointing that out!